include/motor-control/core/utils.hpp

```diff
@@ -17,6 +17,10 @@ auto fixed_point_multiply(sq31_31 a, sq0_31 b) -> sq0_31;
 
 auto fixed_point_multiply(sq31_31 a, uint32_t b) -> uint32_t;
 
+struct radix_offset_0 {};
```

Comment:
Is this a workaround of some sort?

Reply:
Yeah, unfortunately c++ overloaded function dispatch considers `typedef`s to be the same. So `fixed_point_multiply(sq31_31 a, sq0_31 b)` is the same as `fixed_point_multiply(sq31_31 a, int32_t b)`. This dummy struct changes the signature enough that you can specify which function you want, since they are in fact quite different.

Reply:
Do you mind putting a small comment as to why this is needed?